coderedcms/settings.py

```diff
@@ -210,6 +210,10 @@ class _DefaultSettings:
             ),
         ],
         "pagepreviewblock": [
+            (
+                "coderedcms/blocks/miniview_page.html",
```

Comment:
This can probably be omitted... The default (empty string) should be sufficient to trigger this template. In the future I'd like to remove these choices entirely since we have this new functionality.